### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by DianaBeki**
### Milestone

- Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
Inside a transaction:

-  Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
- Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
- Verify that changes were made.
- Commit the transaction.
- Verify that changes persist after commit.
UPDATE

- Now, take a deep breath and... Inside a transaction delete all records in the animals table, then roll back the transaction.

- After the rollback verify if all records in the animals table still exists. After that, you can start breathing as usual ;)
delete

- Inside a transaction:
 -  Delete all animals born after Jan 1st, 2022.
- Create a savepoint for the transaction.
- Update all animals' weight to be their weight 
- Rollback to the savepoint
- Update all animals' weights that are negative to be their weight 
- Commit transaction
delsavecommit

- Write queries to answer the following questions:
-  How many animals are there?
- How many animals have never tried to escape?
- What is the average weight of animals?
- Who escapes the most, neutered or not neutered animals?
- What is the minimum and maximum weight of each type of animal?
- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
queries

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -7,4 +7,37 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
...
+BEGIN;
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL OR species = '';
+SELECT * FROM animals;
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
```
Comment (Munsa1) [inline queries.sql]: - [ ] Here kindly add a select statement after the delete statement and before the rollback statement to verify that the change was made.

e.g
```
SELECT * FROM animals;
```

File: queries.sql [hunk 2]
```diff
@@ -7,4 +7,37 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
...
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT SP1;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO SP1;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg<0;
+ SELECT * FROM animals;
```
Comment (Munsa1) [inline queries.sql:36]: - [ ] Good job verifying the change was made here now please add a `COMMIT;` statement to permanently save all the changes.

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @DianaBeki, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Correct Screenshots :heavy_check_mark: 
Clean Codebase :heavy_check_mark: 
Good Understanding Of Databases :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._
- [ ] Note that the required changes just help improve minor changes to your work to make it perfect, otherwise, this is great work. Keep it up!
### Note Below ⚠️ 
- [ ] Fix the MIT license agreement link as it is broken because you have no license file in your project. Kindly add it
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — APPROVED** [review]
Hi @DianaBeki, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

### Hightlight 
- Added query ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**